README.md

```diff
@@ -1,2 +1,3 @@
-# triggers
-Event triggering with Tekton!
```

Comment:
I think it would be nice to have a one line summary at the top that ppl can easily skim (doesn't have to be this sentence tho!)